pkg/apis/config/v1alpha1/register.go

```diff
@@ -63,8 +64,13 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&AdminQoSConfigurationList{},
 		&AuthConfiguration{},
 		&AuthConfigurationList{},
+		&TransparentMemoryOffloadingConfiguration{},
+		&TransparentMemoryOffloadingConfigurationList{},
 	)
 
+	scheme.AddKnownTypes(workloadapi.SchemeGroupVersion,
```

Comment:
why register TransparentMemoryOffloadingIndicators to workloadapi.SchemeGroupVersion?

Reply:
this extended indicator is used with SPD ExtendedIndicator, it should resides in the same group with SPD.